Source/Chatbook/UI.wl

```diff
@@ -2333,6 +2440,49 @@ currentValueOrigin[
 	]
 ]
 
+(*====================================*)
+
+getModelsMenuItems[] := Module[{
+	items
+},
+	items = Select[
+		Wolfram`Chatbook`Actions`Private`getModelList[],
```

Comment:
It would be better to move `getModelList` to `` Wolfram`Chatbook`Actions` `` rather than reference private contexts like this.